### DISCUSSION TITLE
Chafa 1.12.5

### DISCUSSION THREAD

**Discussion by hpjansson** [hpjansson/chafa#142] [Announcements]
This is a bugfix release.

* Bug fixes:
  #123 (reported by @GuardKenzie).
  #141 (reported by @jstkdng).
  [unfiled] Missing comma in `--help` output (reported by @Freed-Wu).

<hr /><em>This discussion was created from the release <a href='https://github.com/hpjansson/chafa/releases/tag/1.12.5'>Chafa 1.12.5</a>.</em>

---

Another small point release; 1.14 will bring the juicy features, but it will take a bit longer.

**Comment by jstkdng** [comment]
I think the fix for the issue I reported should have been added on 1.14. It'd be easier to check the `CHAFA_VERSION_CUR_STABLE` macro and don't apply the quirk fixes if running that version.

**Reply by hpjansson** [reply]
You can still do that, though, since the quirks presumably are version-agnostic. They will just technically not be needed on 1.12.5.
